.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -1,104 +1,99 @@
 name: Bug Report
 description: Report an issue or unexpected behavior that occurs within the application
-labels: "bug"
+title: "[Bug]: "
+labels: "bug :beetle:"
```

Comment:
This would require the label to be changed on the repo
(not difficult to do but it would ideally be changed shortly after this PR is merged)
![image](https://user-images.githubusercontent.com/78101139/133682202-595ce78c-1fc8-4c90-a714-1b4a466a6367.png) 
Click edit & add the emoji to the label

Reply:
oh yeah no problem we can just delete the label. i just thought it would look nice and that it would somehow know what a beelte is lol